Docs/introducing-blazor-forms.md

```diff
@@ -0,0 +1,215 @@
+# Introducing Blazor Forms
+
+Forms are a key component in any web application as they allow us to collect information from end-users.  This could range from collecting user details as part of a sign-up process through to a single textbox that allows users to leave comments or post messages in an application.
+
+In this article we'll explain the benefits of Blazor forms, we'll look at their lifecycle and we'll explain the fundamental concepts used to create form-driven solutions using Blazor.
+
+## Form Fundamentals
+
+Let's assume that we are building a Contact form which has two fields (Name and Email) and a button to submit the form. The code required to do this in ASP.NET Core would look similar to this.
```

Comment:
We should replace ASP.NET Core with ASP.NET MVC

Reply:
Sure thing. This is fixed now.